### ISSUE TITLE
Dev enhancement: Add IntelliCode Github Action

### ISSUE THREAD

**Issue by WSLUser** [microsoft/terminal#6057]
At MS Build, we discovered a new IntelliCode Github Action has been created. It only requires 2 lines in yaml to enable. Please get this set up to make contributing even easier using Visual Studio! (VS Code is also planned to be supported later).

**Comment by WSLUser** [comment]
Tagging @DHowett for this.

**Comment by DHowett** [comment]
I get e-mails for every single issue that gets filed or commented on in this repository. Much of the team, and many of our followers, are in the same boat. Please don't needlessly tag people to get your issue triaged first.

**Comment by DHowett** [comment]
(Sorry. I am trying not to be grouchy. :smile:) What does this Intellicode action do? I've not yet seen the news!

**Comment by WSLUser** [comment]
Yes sorry! So I think it's best to watch the session itself. But it helps remove duplicate effort, especially once somebody already does some code such as adding a function or expression that will be re-used elsewhere in a file or even a different one in the project. This includes adding exceptions. It will also follow your vars so even if you redefine it later, the IntelliCode will determine where else it exists so you can correct all problems/add functionality everywhere it exists.

**Comment by WSLUser** [comment]
BDL156
IntelliCode – AI-assisted development, now and in the future is the Build session

**Comment by DHowett** [comment]
Excellent. Thanks!

**Comment by WSLUser** [comment]
https://visualstudio.microsoft.com/services/IntelliCode/ also has some information.
